TCAT.xcodeproj/project.pbxproj

```diff
@@ -500,7 +496,7 @@
 				TargetAttributes = {
 					449A7C741D80D0E80019300C = {
 						CreatedOnToolsVersion = 7.3.1;
-						DevelopmentTeam = ZGMCXU7X3U;
+						DevelopmentTeam = 524YAK9BG3;
```

Comment:
@myo3 @AAAstorga Should they be logged into Walker White's account?

Reply:
Yeah probably